docs/en_US/Release.rst

```diff
@@ -110,7 +110,7 @@ Training service
 """"""""""""""""
 
 
-* Support configuring python environment using "preCommand" in remote mode (#2875)
+* Support configuring python environment using "pythonPath" in remote mode (#2875)
```

Comment:
This is the release doc of v1.9, please revert it.

Reply:
Reverted.